akka-http-tests/src/test/scala/akka/http/scaladsl/unmarshalling/sse/LineParserSpec.scala

```diff
@@ -37,11 +40,25 @@ final class LineParserSpec extends AsyncWordSpec with Matchers with BaseUnmarsha
         .map(_ shouldBe Vector("line1", "line2", "line3", "line4", "line5", "line6", ""))
     }
 
+    "ignore a trailing non-terminated line between " in {
```

Comment:
Between what? It's good that this test makes this behavior explicit but I'm not sure it is obvious to me that this is the behavior we want..

Reply:
I think I forgot to finish that sentence in the test case I posted before. Let's call it `"ignore a trailing non-terminated line when parsing byte by byte"`